services/mqtt/config.go

```diff
@@ -0,0 +1,45 @@
+package mqtt
+
+import (
+	"errors"
+	"fmt"
+)
+
+type Config struct {
+	// Enabled indicates whether the service should be enabled
+	Enabled bool `toml:"enabled" override:"enabled"`
+	// URL of the MQTT Broker
+	Host string `toml:"host" override:"host"`
+	// Port of the MQTT Broker
+	Port uint16 `toml:"port" override:"port"`
+
+	ClientID string `toml:"client_id" override:"client_id"`
```

Comment:
Kapacitor uses dashes instead of underscores. Can you update this one and the others in this config struct?